site/docs/api-quickstart.md

```diff
@@ -11,6 +11,7 @@ The Hive catalog connects to a Hive MetaStore to keep track of Iceberg tables. T
 ```scala
 import org.apache.iceberg.hive.HiveCatalog
 
+spark.sparkContext.hadoopConfiguration.set("hive.metastore.warehouse.dir", "path/to/warehouse")
```

Comment:
The temporary path should be a real one. Let's use `s3://bucket/path/to/warehouse`.

Also, can you add a comment about being able to configure Hive here and add a newline between this and instantiating the catalog?

Reply:
Thanks very much for the comment! Addressed.